libass/ass.c

```diff
@@ -321,6 +385,77 @@ static char *next_token(char **str)
     return start;
 }
 
+static int parse_format_line(ASS_FormatLine *fmt, const char *string)
```

Comment:
This function can return `0` or `-1`. Better to replace with `bool`.

Reply:
I think that such "reserved for the future" features are a root of all evil. When that function really needed to differentiate errors then and only then change to `int` should be done.

Reply:
All other functions in this file use classic `int` for error return codes or zero on success, like `main` does. With `bool` you'd normally expect to return `true` on success and false otherwise. Using `bool`as an error indicator only here would only introduce inconsistency in failure checks. *(or create confusion by using `true` as an error indicator)*